docs/pages/dsHDMIIn/ds-hdmi-in_L3_Test-Procedure.md

```diff
@@ -0,0 +1,899 @@
+
+Example Rack configuration File: `ut/host/tests/configs/example_rack_config.yml`
+
+For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/1.0.0/README.md) and [example_rack_config.yml](https://github.com/rdkcentral/python_raft/blob/1.0.0/examples/configs/example_rack_config.yml)
+
+In this file, update the configuration to define the console sessions for the `DUT` and the outbound settings:
+
+|Console Session|Description|
+|---------------|-----------|
+|default|Downloads the streams required for test cases|
+|ssh_player|Plays the stream required for test case|
+|ssh_player_secondary|Plays a secondary stream, if required for test case|
+|ssh_hal_test|Executes the `HAL` binary for the test case|
+
+```yaml
+rackConfig:
+  - dut:
+      ip: "XXX.XXX.XXX.XXX"  # IP Address of the device
+      description: "tv device under test"
+      platform: "tv"
+      consoles:
+        - default:
+            type: "ssh"
+            port: 10022
+            username: "root"
+            ip: "XXX.XXX.XXX" # IP address of the device
+            password: ' '
+        - ssh_player:
+            type: "ssh"
+            port: 10022
+            username: "root"
+            ip: "XXX.XXX.XXX" # IP address of the device
+            password: ' '
+        - ssh_player_secondary:
+            type: "ssh"
+            port: 10022
+            username: "root"
+            ip: "XXX.XXX.XXX" # IP address of the device
+            password: ' '
+        - ssh_hal_test:
+            type: "ssh"
+            port: 10022
+            username: "root"
+            ip: "XXX.XXX.XXX" # IP address of the device
+            password: ' '
+      outbound:
+        download_url: "tftp://tftp-server.com/rack1/slot1/"    # Download location for the CPE device
+        upload_url: "sftp://server-address/home/workspace/tftp/rack1/slot1/" # Upload location
+        upload_url_base_dir: "sftp://server-address/home/workspace/tftp/rack1/slot1"
+        httpProxy:   # Local proxy if required
+        workspaceDirectory: './logs/workspace'   # Local working directory
+
+```
+
+#### Device Configuration File
+
+Example Device configuration File: `ut/host/tests/configs/deviceConfig.yml`
+
+For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/1.0.0/README.md) and [example_device_config.yml](https://github.com/rdkcentral/python_raft/blob/1.0.0/examples/configs/example_device_config.yml)
+
+Update the target directory where `HAL` binaries will be copied into the device. Also, map the profile to the source/sink settings `YAML` file path.
+
+Ensure the platform should match with the `DUT` platform in [Rack Configuration](#rack-configuration-file)
+
+```yaml
+deviceConfig:
+  cpe1:
+    platform: "tv"    # Must match the platform in example_rack_config.yml
+    model: "uk"
+    target_directory: "/tmp"  # Path where HAL binaries are copied in device
+    test:
+      profile: "../../../../profiles/sink/Sink_AudioSettings.yaml"
```

Comment:
mention the proper HDMI input yaml here?

Reply:
done